src/session_stats.cpp

```diff
@@ -44,8 +47,8 @@ namespace libtorrent
 	};
 
 #define METRIC(category, name) { #category "." #name, counters:: name },
-	static const stats_metric_impl metrics[] =
-	{
+	aux::array<stats_metric_impl, counters::num_counters> const metrics
```

Comment:
if you remove ``static`` (which seems reasonable to do), you need to put it in the anonymous namespace